ckan/plugins/interfaces.py

```diff
@@ -1779,3 +1780,14 @@ def before_fork(self):
         u'''
         Called shortly before the CKAN process is forked.
         '''
+
+
+class ICLICommands(Interface):
+    u'''
+    Allow extensions to define click commands.
+    '''
+    def get_commands(self):
```

Comment:
as it called `get_commands` and just in case - maybe it would be better to allow returning list of commands instead of single one from this method? We have a similar experience with `IBlueprints` - it appears that people often want to create multiple entry points inside the extension.

Reply:
Yes, it's definitely better to work with multiple commands at once.